modules/graphman.js

```diff
@@ -1,5 +1,5 @@
 
-const VERSION = "v1.1";
+const VERSION = "v1.2";
 const SCHEMA_VERSION = "v10.1-CR04";
```

Comment:
@uppoju could u please update this schema version to **v11.1.00**

Reply:
ok